lib/course_planner_web/controllers/permissions.ex

```diff
@@ -36,5 +37,10 @@ defimpl Canada.Can, for: CoursePlanner.Accounts.User do
   def can?(%User{id: id}, action, %User{id: id})
     when action in [:show, :edit, :update], do: true
 
+  def can?(_user, action, EventController)
+    when action in [:index, :show], do: true
+
+  def can?(_user, _action, EventController), do: false
```

Comment:
isn't this line already covered in line 45 ?
`def can?(_user, _action, _controller), do: false`

Reply:
Yeah it is. I forgot it's a whitelist approach now.